cypress/integration/splash.spec.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-undef */
+/// <reference types="Cypress" />
+
+describe("home page", () => {

```

Comment:
Just need to change this to splash page.

Reply:
oops, copy paste error